tests/pytests/packaging/test_pkg_download.py

```diff
@@ -0,0 +1,318 @@
+"""
```

Comment:
Let's move this test to be alongside the other package tests?
Then we can move them all when ready into `tests/`?